packages/components-react/projects/nextjs/next.config.js

```diff
@@ -8,8 +8,8 @@ const nextConfig = {
         hostname: 'nav.porsche.com',
       },
       {
-        protocol: 'https',
-        hostname: 'porsche-design-system.github.io',
+        protocol: 'http',
+        hostname: 'localhost',
```

Comment:
**NIT**: Consider explicitly setting `port: 3002` to ensure that the port remains consistent. As I understand it, adding another entry to `remotePatterns` could potentially change the port if it's not explicitly defined.